### PR TITLE
add get email templates for NA message and save NA rating comment

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [Evaluation Form - require comment if answer is NA#170](https://github.com/nerubia/kh360-react/issues/170)

FE PR: [update performance evaluation form to require comment if answer is NA #185](https://github.com/nerubia/kh360-react/pull/185)

### CODE REVIEW

File: src/controllers/user/evaluation_template_contents_controller.ts [hunk 1]
```diff
@@ -15,17 +15,16 @@ export const index = async (req: Request, res: Response) => {
...
-        select: {
-          id: true,
-          name: true,
-          description: true,
-        },
-        where: {
-          evaluation_template_id: evaluation?.evaluation_template_id,
-        },
-      })
+    const evaluationTemplateContents = await prisma.evaluation_template_contents.findMany({
```
Comment (jatnerubia) [inline src/controllers/user/evaluation_template_contents_controller.ts:18]: move all the logic to service

Comment (jatnerubia) [reply src/controllers/user/evaluation_template_contents_controller.ts:18]: also refactor the file name

File: src/controllers/user/user-controller.ts [hunk 2]
```diff
@@ -144,10 +145,17 @@ export const submitEvaluation = async (req: Request, res: Response) => {
   try {
     const user = req.user
     const { id } = req.params
-    const { evaluation_rating_ids, answer_option_ids, comment, is_submitting } = req.body
+    const {
```
Comment (jatnerubia) [inline src/controllers/user/user-controller.ts:148]: move all the logic to service

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

File: src/repositories/evaluation-rating-repository.ts [hunk 3]
```diff
@@ -16,6 +16,12 @@ export const getAllByFilters = async (where: Prisma.evaluation_ratingsWhereInput
   })
 }
 
+export const getFirst = async (where: Prisma.evaluation_ratingsWhereInput) => {
```
Comment (jatnerubia) [inline src/repositories/evaluation-rating-repository.ts:19]: I think `getByFilters` is suitable for this 🤔

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

**Review by jatnerubia — APPROVED** [review]
Cool

**Review by Dev-Nino — APPROVED** [review]

**Pull request merged**